.travis.yml

```diff
@@ -10,5 +10,6 @@ script:
 branches:
     only:
       - master
+      - ammar/course-rerun-import-export
```

Comment:
so that tests in travis can be run on this branch. otherwise tests will run only on master branch.